### PR TITLE
DAOS-7289 test: Enable online drain tests.

### PR DESCRIPTION
**Pull request by rpadma2**
Test-tag: online_drain dmg_negative_test

Signed-off-by: rpadma2 <ravindran.padmanabhan@intel.com>

### CODE REVIEW

**Review by daltonbohning — APPROVED** [review]

**Comment by rpadma2** [comment]
Sigh... I should have made it draft...

**Comment by daltonbohning** [comment]
> Sigh... I should have made it draft...

No worries. I unsubscribed and made it a draft :)

File: src/tests/ftest/osa/online_drain.py [hunk 1]
```diff
@@ -105,16 +106,17 @@ def run_online_drain_test(self, num_pool, oclass=None, app_name="ior"):
...
-            self.run_ior_thread("Read", oclass, test_seq)
-            self.container = self.pool_cont_dict[self.pool[val]][0]
-            kwargs = {"pool": self.pool[val].uuid,
+            self.pool.display_pool_daos_space(display_string)
+            if app_name == "ior":
+                self.run_ior_thread("Read", oclass, test_seq)
+            self.container = self.pool_cont_dict[self.pool][0]
+            kwargs = {"pool": self.pool.uuid,
                       "cont": self.container.uuid}
             output = self.daos_command.container_check(**kwargs)
```
Comment (daltonbohning) [inline src/tests/ftest/osa/online_drain.py:117]: pre-existing / minor - better to use `identifier`, which uses the label if existing. Also, minor, it's not necessary to use kwargs
```suggestion
            output = self.daos_command.container_check(pool=self.pool.identifier, cont=self.container.identifier)
```

**Review by daltonbohning — APPROVED** [review]
LGTM

**Review by shimizukko — APPROVED** [review]

File: src/tests/ftest/osa/online_drain.py (hunk 1 above)
Comment (rpadma2) [reply src/tests/ftest/osa/online_drain.py]: Done. Learnt a new thing :-)

**Review by daltonbohning — APPROVED** [review]

**Review by shimizukko — APPROVED** [review]

**Comment by rpadma2** [comment]
All online drain tests passed: https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-9238/10/testReport/FTEST_osa/OSAOnlineDrain/

**Comment by rpadma2** [comment]
@daos-stack/daos-gatekeeper : This PR is ready for merge.

**Pull request merged**